priv/repo/migrations/20191203121215_create_targets.exs

```diff
@@ -0,0 +1,19 @@
+defmodule TargetMvd.Repo.Migrations.CreateTargets do
+  use Ecto.Migration
+
+  def change do
+    create table(:targets) do
+      add :title, :string
+      add :radius, :integer
+      add :latitude, :float
+      add :longitude, :float
```

Comment:
Postgres provides a daratype for geolocation.
Does phoenix provide a similar interface?

Reply:
There is a package to support that https://github.com/bryanjos/geo_postgis. But I'd rather keep it simple, is PostGIS added by default on Postgres btw?

Reply:
no is not.
you need to install gdal for having that support.
I would consider using it here tough since you will need to perform operations over these coordinates in the future such as knowing the distance between two points or intersections of circumferences.
The math is rather complex if you do it yourself, so using a library for this would be ideal.

Reply:
I looked up a while ago and found [this](https://github.com/pkinney/distance), thought it would be useful

Reply:
I think we can defer the decision to when it's needed. Not a big fan of adding packages before they are needed, plus we can test Ecto's query power. If it gets too cumbersome we can add PostGIS but I'd rather have fewer env dependencies

Reply:
I agree with keeping the dependencies list as thin as possible, but this is a known solution not for a possible problem but for something that's is a card in the project backlog.
Because of that, I consider that using just floats to store the coordinates is not the best option if we know that similar frameworks provide more robust solutions all around.